Dockerfile

```diff
@@ -12,4 +12,4 @@ COPY entrypoint.sh /code/
 COPY import_data.py /code/
 COPY README.md /code/
 RUN chmod +x /code/entrypoint.sh
-CMD ["./entrypoint.sh"]
```

Comment:
Need to remove that `COPY entrypoint.sh /code/` line above! 😜

Reply:
Good catch, I'll fix it!